RebbleOS/vibrate.c

```diff
@@ -48,70 +67,97 @@ void vibrate_init(void)
     _vibrate_queue = xQueueCreate(1, sizeof(uint8_t));
```

Comment:
Ups, nice catch! changed the data size to `sizeof(VibratePattern_t*)`, because I'm passing pointers to `VibratePattern_t` structs, so they are probably `uint32_t`.